src/tutorials/opening-surveyed-data-on-your-computer/index.md

```diff
@@ -2,11 +2,15 @@
 
 [[toc]]
 
-In the last tutorial you learnt how to FIXME.
+In the last tutorial you learnt how to capture field data using <MobileAppName />.
 
-In this tutorial you'll learn how to:
+In this tutorial you'll learn how to transfer data from mobile device to computer in seconds using QGIS, a free and open source desktop GIS package. <MobileAppName /> is based on QGIS, which means it's able to visualise and edit data in the same way QGIS can. This offers us great flexibility which we'll start to see in a moment.
```

Comment:
I think we already have these changes in another PR. so hopefully we do not have conflicts during merging. In general if you work on the same part of the docs, feel free to have one bigger PR, where you mention multiple smaller tickets.